api/service/explorer/structs.go

```diff
@@ -85,38 +101,63 @@ func GetTransaction(tx *types.Transaction, addressBlock *types.Block) *Transacti
 
 // StakingTransaction ...
 type StakingTransaction struct {
-	ID        string   `json:"id"`
-	Timestamp string   `json:"timestamp"`
-	From      string   `json:"from"`
-	To        string   `json:"to"`
-	Value     *big.Int `json:"value"`
-	Bytes     string   `json:"bytes"`
-	Data      string   `json:"data"`
-	GasFee    *big.Int `json:"gasFee"`
-	FromShard uint32   `json:"fromShard"`
-	ToShard   uint32   `json:"toShard"`
-	Type      string   `json:"type"`
+	Transaction
+
+	// CreateValidator
+
+	ValidatorAddress   common.Address          `json:"validator-address"`
```

Comment:
I agree that by expanding fully, we can save a lot of decoding time during access, but this would mean that there will be lot of `null` fields in the output? How would you avoid the possible confusions? 
@Dewansahil?

Reply:
I still think, we could simply store the staking tx specific contents under data (json string) and only populate the to field. Anyways, the explorer is meant to be consumed for information purpose only. Please clarify, @denniswon and @Dewansahil

Reply:
@gupadhyaya agree about the empty fields. that was because of the nature of golang, where it does not allow parent struct to be used as the function return type and have the child struct instances to be returned. Any suggestion? @fxfactorial

Reply:
@gupadhyaya https://stackoverflow.com/questions/24216510/empty-or-not-required-struct-fields-in-golang
Found the solution. will using 'omitempty' tag

Reply:
Since this needs to be discussed further, I will revert this staking transaction fields unification change from this pr to unblock launch issues. Will come back to this discussion later.